pom.xml

```diff
@@ -51,7 +51,7 @@
         <project.build.sourceEncoding>${encoding}</project.build.sourceEncoding>
         <project.reporting.outputEncoding>${encoding}</project.reporting.outputEncoding>
         <licence.acceptPomPackaging>true</licence.acceptPomPackaging>
-        <image.repository/>
+        <image.repository>gchq/palisade-</image.repository>
```

Comment:
remove image.repository from release profile

Reply:
Removed from the release profile